libtransact/src/sawtooth.rs

```diff
@@ -406,7 +406,7 @@ mod xo_compat_test {
         }
     }
 
-    fn create_batch(signer: &Signer, game_name: &str, payload: &str) -> BatchPair {
+    fn create_batch(signer: &HashSigner, game_name: &str, payload: &str) -> BatchPair {
```

Comment:
If possible, we should keep Signer here for example purposes (so if someone is looking at this, its not tied to HashSigner).